application/src/ext/java/org/opentripplanner/ext/debugrastertiles/EdgeVertexTileRenderer.java

```diff
@@ -225,7 +227,9 @@ public void renderTile(TileRenderContext context) {
          * need to expand the envelope by an unbounded amount (max label size).
          */
         double x = tilePoint.getX();
-        if (x + labelWidth > context.tileWidth) x -= labelWidth;
+        if (x + labelWidth > context.tileWidth) {
+          x -= labelWidth;
+        }
```

Comment:
These are the types of changes I like the most in what openrewrite automatically fixed.